### PR TITLE
FIX: fix conda recipe and pip dependencies

### PR DESCRIPTION
**Pull request by mrakitin**
A possible fix for #129.

### CODE REVIEW

**Comment by prjemian** [comment]
So, are we ready to merge?

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -32,19 +32,14 @@ build:
...
-    - bluesky
-    - pyRestTable
-    - pandas
-    - xlrd
+    - pip
   run:
     - python
     - ophyd
-    - databroker
+    - databroker[all]
```
[the remark this answers is not in the document]
Comment (prjemian) [reply conda-recipe/meta.yaml]: fixed in my fork - I see it in your fork now

**Comment by mrakitin** [comment]
@prjemian, it's ready to go. Thanks for all your contributions via https://github.com/mrakitin/apstools/pull/1.

**Comment by mrakitin** [comment]
Awesome, thanks!

**Pull request merged**